uni

```diff
@@ -59,6 +59,34 @@ while [ "$#" -gt 0 ]; do
         fi
     ;;
     --version | -v) printf "${STYLE_LIGHT_CYAN}\n UNI${STYLE_BOLD} v${uni_version}\n\n";;
+    --help | help | -h | h)
+        HELP_INFO="
+${STYLE_LIGHT_CYAN}${STYLE_BOLD}Usage: ${RESET_STYLE}${STYLE_BOLD}uni [OPTIONS]
+
+${STYLE_LIGHT_CYAN}Options:${RESET_STYLE}
+${STYLE_BOLD}  -h, --help${RESET_STYLE}                  Shows available commands
+${STYLE_BOLD}  -v, --version${RESET_STYLE}               Shows the current UNI version installed.
+${STYLE_BOLD}  a, add${RESET_STYLE}                      Adds one or more repositories.
+${STYLE_BOLD}  b, bake ${STYLE_YELLOW}<recipe>${RESET_STYLE}            Cooks one or more given recipes
+${STYLE_BOLD}  b how, bake how ${STYLE_YELLOW}<recipe>${RESET_STYLE}    Shows what one or more recipes does
+${STYLE_BOLD}  i, install ${STYLE_YELLOW}<package>${RESET_STYLE}        Installs one or more packages
+${STYLE_BOLD}  u, update${RESET_STYLE}                   Updates the package lists from your repositories
+${STYLE_BOLD}  up, upgrade${RESET_STYLE}                 Upgrades deeply the system and remove useless dependencies
+${STYLE_BOLD}  up-me, upgrade-me${RESET_STYLE}           Looks for UNI upgrades, if available it will be upgraded
+${STYLE_BOLD}  rm, remove ${STYLE_YELLOW}<package>${RESET_STYLE}        Removes one or more packages
+${STYLE_BOLD}  rm-me, remove-me${RESET_STYLE}            Removes UNI from the system
+
+${STYLE_LIGHT_CYAN}Available Recipes (by Dalton Menezes):${RESET_STYLE}
+${STYLE_YELLOW}  atom-install${RESET_STYLE}                Installs the latest version of Atom Editor
+${STYLE_YELLOW}  telegram-install${RESET_STYLE}            Installs the latest version of Telegram
+${STYLE_YELLOW}  node-install${RESET_STYLE}                Installs the nodejs latest LTS available version and npm
+${STYLE_YELLOW}  yarn-install${RESET_STYLE}                Installs the latest version of Yarn
+${STYLE_YELLOW}  stylishbuntu-theme${RESET_STYLE}          Theme Package
+${STYLE_YELLOW}  nebulosa-theme${RESET_STYLE}              Theme Package
+${STYLE_YELLOW}  mac-theme${RESET_STYLE}                   Theme Package
```

Comment:
This shouldn't be hard-coded. Can you fix this bringing this information from GitHub?
In this case, it should fetch from [here](https://github.com/uni-linux/recipes/tree/master/src/daltonmenezes)

Reply:
I've updated to grab the README from that github with minor transformation for readability